erigon-lib/diagnostics/snapshots.go

```diff
@@ -2,17 +2,18 @@ package diagnostics
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
 	"time"
 
+	"github.com/ledgerwatch/erigon-lib/common"
 	"github.com/ledgerwatch/erigon-lib/kv"
 	"github.com/ledgerwatch/log/v3"
 )
 
 var (
 	SnapshotDownloadStatisticsKey = []byte("diagSnapshotDownloadStatistics")
 	SnapshotIndexingStatisticsKey = []byte("diagSnapshotIndexingStatistics")
+	SnapshotFillDBStatisticsKey   = []byte("diagSnapshotFillDBStatistics")
```

Comment:
this data stored in own db and in own table, probably no reason to have `diagSnapshot` prefix in this case.